python/common/org/Python.java

```diff
@@ -384,12 +384,12 @@ public static void initializeModule(java.lang.Class cls, java.util.Map<java.lang
             while (true) {
                 org.python.Object next = iter.__next__();
                 if (!next.toBoolean()) {
-                    return new org.python.types.Bool(false);
+                    return org.python.types.Bool.getBool(false);
```

Comment:
In the case where you're referencing a constant, is there any reason to not just use the static member reference directly? Variable access should always be faster than a function call...

Reply:
Ah, yes! Done (Here and many other places)